client/src/actions/profileActions.js

```diff
@@ -109,16 +109,12 @@ export const deleteAccount = history => dispatch => {
       .delete("/api/remove")
       .then(res => {
         history.push("/");
-        dispatch({
-          type: SET_CURRENT_USER,
-          payload: {}
-        });
         dispatch(logoutUser());
       })
       .catch(err => {
         dispatch({
           type: GET_ERRORS,
-          payload: err.response.data
+          payload: {},
```

Comment:
why clearing errors here?

Reply:
When u delete the account nothing should be there try delete the account in ur branch  we are getting error.

Reply:
When deleting account nothing should be there in redux. Try deleting in your local version of code getting error. and we are login out.

Reply:
yeah, but getting error means something happened to `delete account`. `err` should always be there.